index.html

```diff
@@ -210,5 +210,48 @@ <h3>CSS</h3>
         </section>
       </div>
     </div>
+
+    <section class="form-container" id="contact-me-id">
+      <div class="section">
+        <div>
+          <h2 class="form-heading">Contact me</h2>
+          <p class="form-para">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+
+        <form action="https://formspree.io/f/xlevolgp" method="post">
+          <label for="name"></label>
+          <input
+            name="name"
+            id="name"
+            type="text"
+            placeholder="name"
+            maxlength="30"
+            required
+          />
+
+          <label for="email"></label>
+          <input
+            name="Email"
+            id="email"
+            type="email"
+            placeholder="enter your email"
+            required(30)
+          />
+          <textarea
+            name="textarea"
+            id="textarea"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            placeholder="Write your message here"
```

Comment:
- According to the project requirements, all input fields should have the `required` attribute. Please fix this 🙏

Reply:
Please @mckent05 according to the project requirement, we're not required to use media queries on the mobile-version

Can you be more specific with your explanations? my project is responsive in all three mobile versions (mobile-small, mobile-medium, and mobile-large) and I used flex-box properties to work on it.